environment.yml

```
@@ -34,8 +34,8 @@ dependencies:
 - pyyaml~=5.1.2
 - reentry~=1.3
 - simplejson~=3.16
-- sqlalchemy-utils~=0.36.0
-- sqlalchemy~=1.3.10
+- sqlalchemy-utils~=0.37.0
+- sqlalchemy~=1.3
```

```suggestion
- sqlalchemy~=1.3.0|~=1.4.0
```
We have to constrain sqlachemy to specific minor versions since the project follows a [modified semantic versioning scheme](https://www.sqlalchemy.org/download.html#versions) where breaking changes can be introduced with new minor versions.